components/perl/xml-sax-base/xml-sax-base-PERLVER.p5m

```diff
@@ -0,0 +1,37 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Aurelien Larcher
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-$(PLV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="XML::SAX::Base - Base class for SAX Drivers and Filters"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/perl5/$(PERLVER)/man/man3/XML::SAX::Base.3
+file path=usr/perl5/$(PERLVER)/man/man3/XML::SAX::BuildSAXBase.3
+file path=usr/perl5/$(PERLVER)/man/man3/XML::SAX::Exception.3
+file path=usr/perl5/vendor_perl/$(PERLVER)/XML/SAX/Base.pm
+file path=usr/perl5/vendor_perl/$(PERLVER)/XML/SAX/BuildSAXBase.pl
+file path=usr/perl5/vendor_perl/$(PERLVER)/XML/SAX/Exception.pm
+file path=usr/perl5/vendor_perl/$(PERLVER)/$(PERL_ARCH)/auto/XML/SAX/Base/.packlist
+
+depend type=conditional fmri=$(COMPONENT_FMRI)-$(PLV) \
```

Comment:
Does this line belong to GENFRAG?

Reply:
Not sure how it works but that's how it is done in the solaris-userland recipe.

Reply:
It's clearly wrong in solaris-userland.
We have *-PERLVER package, which should just depend on library/perl-5/xml-sax-base (which is stated in comment) 
and *-GENFRAG package which should depend on any version depending on installed perl version (this is what this dependency actually means)

Reply:
If runtime/perl-$(PLV) is installed, depend on library/perl-5/xml-sax-base-$(PLV)

Reply:
A similar component would be https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/components/perl/DBI/DBI-PERLVER.p5m , should I do the same?

Reply:
No Perl component actually seem to declare a GENFRAG manifest.

Reply:
Yes, it is generated correctly.